app/views/maneuvers/index.haml

```diff
@@ -1,6 +1,3 @@
-%h1 Which maneuver are you judging?
```

Comment:
Decide to do away with this question entirely? I don't disagree, just wondered if it was intentional.

Reply:
Yeah, the voice seemed pretty different from the other page titles.

Reply:
Agreed. I did everything one way, but then wanted this page to seem friendlier (you may recall that we had some concerns about less-tech-savvy users being unwilling to use mobile devices at all).